src/data/markdown/docs/02 javascript api/07 k6-experimental/01 browser.md

```diff
@@ -86,9 +86,17 @@ $ k6 run script.js
 ```
 
 ```bash
-# When using the `k6:master-with-browser` Docker image, you need to add `--cap-add=SYS_ADMIN`
-# to grant further system permissions on the host for the Docker container.
-docker run --rm -i --cap-add=SYS_ADMIN grafana/k6:master-with-browser run - <script.js
+# WARNING!
+# Be aware that grafana/k6:master-with-browser launches Chrome browser by setting the
+# 'no-sandbox' argument. Use it only with trustworthy websites.
+#
+# As an alternative, our recommendation is to use a Docker SECCOMP profile instead, and
+# overwrite the Chrome arguments to not use 'no-sandbox' such as:
```

Comment:
```suggestion
# overwrite the Chrome arguments not to use 'no-sandbox' such as:
```

Reply:
I believe both of these forms are accepted. In this case, using `not to use 'no-sandbox'` kind of sounds worse to me than the initial option :thinking: But no strong opinion. Any feedback on this @heitortsergent ?

Reply:
Yeah, I have no strong opinion, but there are subtle semantic differences. I'm resolving this :)